### ISSUE TITLE
BatchNorm and DropOut layer control - Train vs Eval mode 

### ISSUE THREAD

**Issue by NarasimhaM3101** [tensorflow/ranking#30]
Can someone please clarify if batchnorm and drop out layer modes are automatically switched or we have to manually control train vs eval mode. 

I presume for custom tf.estimators, we have to add tf.control_dependencies. Do we have to do this tensorflow ranking as well? 

Please clarify with an example if you could. 

Thanks,
Narasimha

**Comment by ramakumar1729** [comment]
Yes, you would have to add control dependencies in the same way as for tensorflow.

**Comment by xuanhuiwang** [comment]
Closing this.
